src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+function Calculator() {
+  return (
+    <div className="top-div">
+      <div className="zero-div"><div className="num-item  top-num">0</div></div>
+      <div className="numerals">
+        <div className="num-item">AC</div>
```

Comment:
- [ ] As instructed in the requirement, please kindly ensure that this component has atleast on child component. This [tutorial might be helpful](https://medium.com/@martin.crabtree/react-js-using-children-props-c83d5b259756). 🙏